scripts/packages/postinstall.sh

```diff
@@ -249,37 +227,33 @@ api:
   port: 8081
 EOF
     printf "PostInstall: Updating file permissions for nginx-agent.conf to 0640\n"
-    chmod 0640 "${BSD_HIER}"/etc/nginx-agent/nginx-agent.conf
+    chmod 0640 /etc/nginx-agent/nginx-agent.conf
     fi
 }
 
 upgrade_config_file() {
```

Comment:
I think you can remove the functions `upgrade_config_file` and `add_default_config_file`

Reply:
The default `nginx-agent.conf` is packaged with the agent so I actually dont know why these functions are here. Its probably for some legacy reasons. I'd say remove them and then test out the default install to make sure things still work.